components/dashboard/src/projects/ConfigureProject.tsx

```diff
@@ -244,9 +227,8 @@ export default function () {
         redirectToNewWorkspace();
     }
 
-    return <>
-        <Header title="Configuration" subtitle="View and edit project configuration." />
-        <div className="app-container mt-8 flex space-x-4">
+    return <PageWithSubMenu subMenu={getProjectSettingsMenu(project, team)} title="Configuration" subtitle="View and edit project configuration.">
+        <div className="flex space-x-4">
             <div className="flex-1 h-96 rounded-xl overflow-hidden relative flex flex-col">
                 <div className="flex bg-gray-50 dark:bg-gray-800 border-b border-gray-200 dark:border-gray-600 px-6 pt-3">
                     <TabMenuItem name=".gitpod.yml" selected={selectedEditor === '.gitpod.yml'} onClick={() => setSelectedEditor('.gitpod.yml')} />
```

Comment:
**question**: I could be missing something but do we think it makes sense to remove the latest prebuild block here and only keep the configuration n now that we've removed this from the onboarding flow? Later on we could also handle cases where no configuration is added yet.

**thought**: Eventually we will need to separate project addition and project configuration steps to that these can be done separastely.

Reply:
Sure, I'm happy to remove the "Prebuild Runner", i.e. the right part of the split screen, and keep only the editor.

However, this feels like a follow-up issue (because I think we also need to fix the copy).